example_conf/developer.example.conf

```diff
@@ -0,0 +1,42 @@
+
+explain = "copy the directory and change the file names to developer.conf"
```

Comment:
rename this file to developer.conf (The exampleness of it is evident from the parent directory)